### PR TITLE
fix: instance management doc

### PR DESCRIPTION
**Pull request by jnikhila**
- Structure changes
- Rewriting

### CODE REVIEW

**Review by danciaclara — APPROVED** [review]

File: website/docs/getting-started/setup/instance-management/README.mdx [hunk 1]
```diff
@@ -6,18 +6,57 @@ import Tabs from '@theme/Tabs';
...
-
-This brings down the `auto_update` container, and update checks are no-longer performed. Note that however, if you run `docker-compose up -d` later, for any reason, then this `auto_update` will be brought up again. You can use that to turn auto updates on again in the future.
-
-To check if auto updates are turned on in your instance, please run `docker-compose ps` and see if there a server called `auto_update` listed in the output, and if it's status is `Up`. If not, then auto updates are turned off for your instance.
-
-## Further reading
-* [Update Appsmith](/getting-started/setup/instance-management/update-appsmith)
+One of the key features of Appsmith is its instance management capabilities. This page is a gateway to various management tasks you can perform on your Appsmith instance. 
+
+You can manage your instance by performing:
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/README.mdx:11]:
```suggestion
You can manage your instance by:
```

File: website/docs/getting-started/setup/instance-management/README.mdx [hunk 2]
```diff
@@ -6,18 +6,57 @@ import Tabs from '@theme/Tabs';
...
+* Backups for Appsmith instance and database
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/README.mdx:12]:
```suggestion
* Creating backups for the Appsmith instance and database
```

File: website/docs/getting-started/setup/instance-management/README.mdx [hunk 3]
```diff
@@ -6,18 +6,57 @@ import Tabs from '@theme/Tabs';
...
+* Updating Appsmith to a new version
+* Scheduling a maintenance window for auto updates
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/README.mdx:14]:
```suggestion
* Scheduling a maintenance window for automatic updates
```

File: website/docs/getting-started/setup/instance-management/README.mdx [hunk 4]
```diff
@@ -6,18 +6,57 @@ import Tabs from '@theme/Tabs';
...
+* Restoring backups.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/README.mdx:15]:
```suggestion
* Restoring backups
```

File: website/docs/getting-started/setup/instance-management/update-appsmith.mdx [hunk 5]
```diff
@@ -168,6 +168,107 @@ docker-compose up -d
 If you haven't been using your Appsmith instance for a while, it could be possible that there might be multiple versions that show up in the error message. In such a case, perform the same steps for each version in order, starting from the oldest.
 :::
 
+## Switch off auto updates
```
Comment (sum35h) [inline website/docs/getting-started/setup/instance-management/update-appsmith.mdx:171]:
```suggestion
## Disable auto updates
```

File: website/docs/getting-started/setup/instance-management/update-appsmith.mdx [hunk 6]
```diff
@@ -168,6 +168,107 @@ docker-compose up -d
...
+
+ Follow the below steps for your Appsmith installation to switch off auto updates:
```
Comment (sum35h) [inline website/docs/getting-started/setup/instance-management/update-appsmith.mdx:173]:
```suggestion
 Follow the below steps for your Appsmith installation to disable auto updates:
```

File: website/docs/getting-started/setup/instance-management/update-appsmith.mdx [hunk 7]
```diff
@@ -168,6 +168,107 @@ docker-compose up -d
...
+  ```
+  and run the below command:
+  ```bash
+    helm upgrade --values values.yaml appsmith appsmith/appsmith
+  ```
+
+</TabItem>
+<TabItem label="Business Edition" value="businessEdition"> 
+
+You can switch off auto update by either using a command or setting it in the `values.yaml` file.
```
Comment (sum35h) [inline website/docs/getting-started/setup/instance-management/update-appsmith.mdx:243]:
```suggestion
You can disable auto update by either using a command or setting it in the `values.yaml` file.
```

File: website/docs/getting-started/setup/instance-management/update-appsmith.mdx [hunk 8]
```diff
@@ -168,6 +168,107 @@ docker-compose up -d
...
+     depends_on:
+       - appsmith
+     environment:
+       - WATCHTOWER_LIFECYCLE_HOOKS=true
+   ```
+
+3. Save the file and run:
+
+```bash
+docker-compose up -d
```
Comment (sum35h) [inline website/docs/getting-started/setup/instance-management/update-appsmith.mdx:203]:
```suggestion
docker-compose up -d --force-recreate
```

**Review by sum35h — APPROVED** [review]

File: website/docs/getting-started/setup/instance-management/update-appsmith.mdx (hunk 7 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-management/update-appsmith.mdx:243]: As per the style guide, disable is considered a strong word and should be avoided so keeping it as is.

**Review by danciaclara — APPROVED** [review]

**Pull request merged**